main/ejml-dsparse/src/org/ejml/sparse/csc/RandomMatrices_DSCC.java

```diff
@@ -293,4 +350,13 @@ public static void ensureNotSingular( DMatrixSparseCSC A, Random rand ) {
             A.set(s[col], col, rand.nextDouble() + 0.5);
         }
     }
+
+    /**
+     *
+     */
+    public enum EntryDistribution{
```

Comment:
This isn't being used yet correct? Could you hold off on adding it until it's used in code?

Reply:
Right. I also didnt plan to commit it. Slipped through somehow.